### ISSUE TITLE
using the latest version of tensorflow

### ISSUE THREAD

**Issue by countback** [yangsaiyong/tf-adaptive-softmax-lstm-lm#2]
thanks a lot for your code, it seem that your code was writed using Tensorflow under version 0.11 or 0.12, if I want to using the latest version of tensorflow, how I need to modify the code?

**Comment by yangsaiyong** [comment]
Code updated, it can run correctly under tensorflow r1.0
